src/tests/api/Filter.test.js

```diff
@@ -0,0 +1,136 @@
+import { applyFilter } from 'src/api';
+import { filters } from './data-mocks/filters';
+
+const makePossibleFeature = (value) => [
+  {
+    type: 'Feature',
+    geometry: {
+      type: 'Point',
+      coordinates: [0, 0]
+    },
+    properties: {
+      column1: value
+    }
+  },
+  {
+    column1: value
+  }
+];
+
+describe('Filters', () => {
+  test('should return 1 if no filters present', () => {
+    const params = { filters: {}, type: 'number' };
+    expect(applyFilter(params)()).toBe(1);
+  });
+
+  test('should return true if no filters present', () => {
+    const params = { filters: {}, type: 'boolean' };
+    expect(applyFilter(params)()).toBe(true);
+  });
+
+  describe('feature passes filter - boolean type', () => {
+    const params = { filters, type: 'boolean' };
```

Comment:
I find it a bit difficult to think about the test results, having filters imported from an external file, instead of declaring them inline. If feature building is outside the test and also the filters... it's a bit too abstract to me, as I have to navigate between 3 files to understand what's happening...

Reply:
Even if same mocks can be reused in 2 different set of tests... I would probably add them in the same test file for clarity